vagrant/Vagrantfile

```diff
@@ -90,7 +93,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             pm2 logs > /home/vagrant/log/"#{ENV['REPO_NAME']}"/vagrant.log &
           fi
           waitForPM2Daemon
+          cp /home/vagrant/src/build-config/vagrant/rackhd-pm2-config*.yml /home/vagrant/.
+          if (echo "#{ENV['REPOS_UNDER_TEST']}"|grep -q "image-service"); then
+            cp rackhd-pm2-config-image-service.yml rackhd-pm2-config.yml
```

Comment:
Thanks Alan for the comment, but I'm afraid they couldn't be combined as anyway we need the rackhd-pm2-config.yml and rackhd-pm2-config-image-service.yml.

That is:
if combining into lie 98: we can use if image-service,
 cp /home/vagrant/..../rackhd-pm2-config-image-service.yml /home/vagrant/.
And then the if clause have to be followed by an else:
cp /home/vagrant/.../rackhd-pm2-config.yml /home/vagrant/.